cloudigrade/util/misc.py

```diff
@@ -1,11 +1,31 @@
 """Miscellaneous utility functions."""
 import datetime
 import logging
+import re
 from contextlib import contextmanager
 
 from django.db import transaction
 
 logger = logging.getLogger(__name__)
+REDACT_KEY_PATTERN = ".*(password|secret|cert|key).*"
```

Comment:
We're missing the pattern for `rdsCa` in database, `clientAccessToken` in featureFlags,  all the others are matched with the regex.